sysl2/sysl/seqs/visitor.go

```diff
@@ -210,7 +237,23 @@ func (v *SequenceDiagramVisitor) Visit(e Element) error {
 
 	switch t := e.(type) {
 	case *EndpointCollectionElement:
-		return v.visitEndpointCollection(t)
+		v.visitEndpointCollection(t)
+		for bbKey, bbVal := range t.blackboxes {
+			if bbVal.valueType >= BB_APPLICATION && bbVal.visitCount == 0 {
+				var scope string
+				switch bbVal.valueType {
+				case BB_APPLICATION:
+					scope = "'" + v.currentApp + " :: " + "'"
+				case BB_ENDPOINT_COLLECTION:
+					scope = "'" + v.currentApp + " :: " + t.title + "'"
+				default:
+					scope = ""
+					log.Warnf("blackbox" + scope + "not hit: " + bbKey + "\n")
+				}
+				v.stdout.Write([]byte("blackbox '" + bbKey + "' not hit in app " + scope + "\n"))
```

Comment:
Why is this going to `v.stdout`?

Reply:
It's needed for log validation as part of test case.